Source/PropertyExtraction.swift

```diff
@@ -24,11 +24,16 @@
 
 import Foundation
 
-infix operator <-! { associativity left precedence 140 }
-infix operator <-? { associativity left precedence 140 }
+precedencegroup DictionaryExtractionPrecedence {
+    associativity: left
+    higherThan: AssignmentPrecedence
+}
+
+infix operator <-! : DictionaryExtractionPrecedence
+infix operator <-? : DictionaryExtractionPrecedence
 
-infix operator <--! { associativity left precedence 140 }
-infix operator <--? { associativity left precedence 140 }
+infix operator <--! : DictionaryExtractionPrecedence
+infix operator <--? : DictionaryExtractionPrecedence
```

Comment:
This is pretty cool. I would say normally the `:` should be against the declaration, but that probably doesn't work here.